server/src/main/java/org/apache/uniffle/server/storage/LocalStorageManager.java

```diff
@@ -64,17 +70,34 @@ public class LocalStorageManager extends SingleStorageManager {
       throw new IllegalArgumentException("highWaterMarkOfWrite must be larger than lowWaterMarkOfWrite");
     }
     for (String storagePath : storageBasePaths) {
-      localStorages.add(LocalStorage.newBuilder()
-          .basePath(storagePath)
-          .capacity(capacity)
-          .lowWaterMarkOfWrite(lowWaterMarkOfWrite)
-          .highWaterMarkOfWrite(highWaterMarkOfWrite)
-          .shuffleExpiredTimeoutMs(shuffleExpiredTimeoutMs)
-          .build());
+      try {
+        LocalStorage localStorage = localStorageBuilder
+            .basePath(storagePath)
+            .capacity(capacity)
+            .lowWaterMarkOfWrite(lowWaterMarkOfWrite)
+            .highWaterMarkOfWrite(highWaterMarkOfWrite)
+            .shuffleExpiredTimeoutMs(shuffleExpiredTimeoutMs)
+            .build();
+        localStorages.add(localStorage);
+      } catch (RuntimeException exception) {
+        LOGGER.error("Initializing the storage path of {} failed.", storagePath, exception);
+      }
     }
+    if (localStorages.size() <= 0) {
```

Comment:
Should we introduce a configOption  that if the broken disks are more than the value, the start shuffle will fail to start.

Reply:
Make sense.